core/src/main/java/org/bitcoinj/evolution/SimplifiedMasternodeListManager.java

```diff
@@ -450,12 +450,14 @@ public boolean isQuorumRotationEnabled(LLMQParameters.LLMQType type) {
         return params.getLlmqDIP0024InstantSend() == type && quorumRotationActive;
     }
 
-    // TODO: this needs an argument for LLMQType
     public SimplifiedMasternodeList getListAtChainTip() {
-        return getMasternodeList();
+        if (quorumState.getMasternodeListAtTip() != null) {
+            return quorumState.getMasternodeListAtTip();
+        } else {
+            return quorumRotationState.getMnListAtH();
+        }
     }
```

Comment:
This helps fix a sync issue.  When checking if `mnlistdiff` needs to be requested, we were actually checking the wrong list from `qrinfo`.  Additionally in Dash Wallet, the block height used for the CrowdNode APY was from `qrinfo` which could be up to 12 hours old, rather than the most recent masternode list block height.